examples/src/main/scala/Simple.scala

```diff
@@ -79,5 +80,6 @@ object SimpleMain extends App {
   finally {
     transport.shutdown()
     server()
+    pool.shutdown()
```

Comment:
well, this is a thread pool that they are giving to us, we shouldn't shut down their thread pool

whether or not having them give us a thread pool instead of just a number of threads is perhaps another question.  A question which becomes moot when we move to netty4